core/server/api/canary/utils/serializers/input/members.js

```diff
@@ -0,0 +1,50 @@
+const _ = require('lodash');
+const debug = require('ghost-ignition').debug('api:canary:utils:serializers:input:members');
+const utils = require('../../index');
+
+function defaultRelations(frame) {
+    if (frame.options.withRelated) {
+        return;
+    }
+
+    if (frame.options.columns && !frame.options.withRelated) {
+        return false;
+    }
+
+    frame.options.withRelated = ['labels'];
+}
+
+module.exports = {
+    browse(apiConfig, frame) {
+        debug('browse');
+
+        if (!utils.isContentAPI(frame)) {
```

Comment:
Do we allow for Content API calls to the members endpoints? :thinking:

Reply:
We don't actually, probably this is again an overkill and hangover of mimicing post/tag behavior where this is needed. I'll clean it up, thanks 😄